### DISCUSSION TITLE
socket: too many open files, retrying with remaining

### DISCUSSION THREAD

**Discussion by onurdrsn** [projectdiscovery/nuclei#2040] [Q&A] [answered]
This is my problem this time

                           __     _
         ____  __  _______/ /__  (_)
        / __ \/ / / / ___/ / _ \/ /
       / / / / /_/ / /__/ /  __/ /
      /_/ /_/\__,_/\___/_/\___/_/   2.7.1
      
		      projectdiscovery.io
      
      [WRN] Use with caution. You are responsible for your actions.
      [WRN] Developers assume no liability and are not responsible for any misuse or damage.
      [INF] Using Nuclei Engine 2.7.1 (latest)
      [INF] Using Nuclei Templates 9.0.2 (latest)
      [INF] Templates added in last update: 3541
      [INF] Templates loaded for scan: 68
      [ERR] Could not register to oast.me: could not make register request: POST https://oast.me/register giving up after 1 attempts: Post "https://oast.me/register": dial tcp 178.128.209.14:443: i/o timeout, retrying with remaining
      [ERR] Could not register to oast.me: could not make register request: POST https://oast.site/register giving up after 1 attempts: Post "https://oast.site/register": dial tcp: lookup oast.site on [2a02:ff0:4:327:212:154:100:18]:53: dial udp [2a02:ff0:4:327:212:154:100:18]:53: socket: too many open files, retrying with remaining
      [ERR] Could not register to oast.me: could not make register request: POST https://oast.live/register giving up after 1 attempts: Post "https://oast.live/register": dial tcp: lookup oast.live on [2a02:ff0:4:327:212:154:100:18]:53: dial udp [2a02:ff0:4:327:212:154:100:18]:53: socket: too many open files, retrying with remaining
      [ERR] Could not register to oast.me: could not make register request: POST https://oast.fun/register giving up after 1 attempts: Post "https://oast.fun/register": dial tcp: lookup oast.fun on [2a02:ff0:4:327:212:154:100:18]:53: dial udp [2a02:ff0:4:327:212:154:100:18]:53: socket: too many open files, retrying with remaining
      [ERR] Could not register to oast.me: could not make register request: POST https://oast.pro/register giving up after 1 attempts: Post "https://oast.pro/register": dial tcp: lookup oast.pro on [2a02:ff0:4:327:212:154:100:18]:53: dial udp [2a02:ff0:4:327:212:154:100:18]:53: socket: too many open files, retrying with remaining
      [ERR] Could not register to oast.me: could not make register request: POST https://oast.online/register giving up after 1 attempts: Post "https://oast.online/register": dial tcp: lookup oast.online on [2a02:ff0:4:327:212:154:100:18]:53: dial udp [2a02:ff0:4:327:212:154:100:18]:53: socket: too many open files, retrying with remaining
      [ERR] Could not initialize interactsh client: could not create client: could not register to servers: could not make register request: POST https://oast.online/register giving up after 1 attempts: Post "https://oast.online/register": dial tcp: lookup oast.online on [2a02:ff0:4:327:212:154:100:18]:53: dial udp [2a02:ff0:4:327:212:154:100:18]:53: socket: too many open files

**Comment by ehsandeep** [comment, marked as answer]
From https://github.com/projectdiscovery/nuclei/issues/2041#issuecomment-1134738318

> This basically means there weren't available file descriptors to use at the time you tried to open a connection to oast.me to register. Was the machine busy doing heavy network scans at that time? IF so, that might be the issue. If not, please share the sudo ulimit -a results to better understand how many open file descriptors your system is configured to support.

**Comment by ehsandeep** [comment]
@onur55-tr this looks like an issue with the system in use where no file descriptors are available to use as suggested by @jimen0
